yklstartdeepdoc.py

```
@@ -0,0 +1,1095 @@
+                    "saved_files": [str(s) for s in saved],
+                }
+
+        except Exception as e:
+            logger.error(f"解析 {pdf_path.name} 时出错: {e}")
+            traceback.print_exc()
+            return None
+
+    def parse_pdf_legacy(self, pdf_path, output_dir=None):
+        """
+        使用传统的 PdfParser.__call__() 解析 PDF
+        保留向后兼容性，支持图像提取
```

Ruff (0.16.1)

[warning] Docstring contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF002)